### ISSUE TITLE
Nested import failed to be fetch

### ISSUE THREAD

**Issue by davinkevin** [mobilexag/plugin-sass#14]
Hi, 

I have a specific behavior and I would like to see if it's a bug.

I have a project with a JS file importing one scss in the folder /config/style/bootstrap.js : 

```js
import angular from 'angular';
import './bootstrap.scss!';

export default angular.module('app.config.bootstrap', []);
```

And in my ./bootstrap.scss I have the following : 
```scss
@import "myvar";
@import "/jspm_packages/github/twbs/bootstrap-sass@3.3.5/assets/stylesheets/bootstrap";
```

The file _myvar.scss is correctly loaded like the file _bootstrap.scss located in jspm_packages.
But the file _bootstrap.scss contains some relative ```@import``` too (https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/_bootstrap.scss).

The result is the sass plugin tries to load all the file relative to the location of my JS instead of using the jspm_package path.

The result of the console is the following : 
![image](https://cloud.githubusercontent.com/assets/1970922/10783056/37c3f780-7d55-11e5-9d5d-5f28c80fcc1f.png)

You can see the folder tree of the project (i'm serving from /public and ./ when running in browser-sync)
![image](https://cloud.githubusercontent.com/assets/1970922/10783075/66bd1c10-7d55-11e5-9676-c5ba76c9622b.png)

I don't know I to handle this problem, could you help ?

Thanks

PS : I've changed some folder name to simplify the example (and avoid having multiple bootstrap folder at different location, this is why in the github project I have another folder name for the bootstrap.js)

**Comment by screendriver** [comment]
I believe you have to modify [paths](https://github.com/systemjs/systemjs/blob/master/docs/config-api.md#paths). Maybe @phenomnomnominal knows a trick for this?

**Comment by JustinWilabyPrecor** [comment]
I've tracked this problem down to 2 issues in `sass-inject.js` 

`urlBase` is being set and persisted on line 64 then used on line 29.  All subsequent calls to load partials use the `urlBase` of the parent scss.

The result is that if I have `/scss/modules/module1.scss` that imports partials from `/scss/partials/_variables.scss`, sass-inject will set the `urlBase=/scss/modules/` when loading `module1.scss` and use it when looking for the partials which will not exist there. e.g. `_variables.scss` will be looked for in `/scss/module/_variables.scss`

**Comment by screendriver** [comment]
Any pull requests to fix this are welcome :sunglasses:

**Comment by JustinWilabyPrecor** [comment]
@davinkevin - Can you confirm PR #21 fixes your issue?  My tests pass but another set of eyes is helpful.  Thanks!

**Comment by dougludlow** [comment]
@screendriver I believe I've got a fix for the issue, if you wouldn't mind reviewing.

**Comment by screendriver** [comment]
Fixed with pull request #24
